examples/analog_hamiltonian_simulation/03_Parallel_Jobs_on_Aquila.ipynb

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
<p>A title for the plot would be great. This can help compare results between QPU and simulation.</p><p> </p><p>Also, You can put the two plots side by side with <code>matplotlib.pyplot.subplots(1,2 ...)</code> , that will make it even easier to compare. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>